src/test/scala/com.snowplowanalytics.snowplow.micro/MicroSpec.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.micro
+
+import org.specs2.mutable.Specification
+
+class MicroSpec extends Specification {
+  "Micro" >> {
+    // This test is to ensure that if the default collector config, located at
+    // `src/main/resources/application.confg` is invalid, the tests will not pass
+    "will throw an exception if the default collector config is invalid" >> {
+      ConfigHelper.parseConfig(Array(
```

Comment:
Hey @greg-el I had second thoughts about this test.

Isn't it better to test the case of an empty array?  That's the case that represents running micro without providing any configuration

```
ConfigHelper.parseConfig(Array()) must(throwA[Exception])
```

Reply:
To explain my comment... the ConfigHelper uses `application.conf` under the hood -- but that's an implementation detail of how it works.  The real requirement of ConfigHelper is that it returns a valid parsed config even if you call it with no arguments.  So we should explicitly test that case, instead of tricking it with a file which doesn't make sense to pass in anyway.

Reply:
I agree, that makes more sense to test. 

I've also renamed the file from `MicroSpec` to `ConfigHelperSpec` to reflect what the file contains better.